var/spack/repos/builtin/packages/libbacktrace/package.py

```diff
@@ -17,7 +17,14 @@ class Libbacktrace(AutotoolsPackage):
     version("master", branch="master")
     version("2020-02-19", commit="ca0de0517f3be44fedf5a2c01cfaf6437d4cae68")
 
+    variant("shared", default=False, description="Additionally build shared library")
+
     depends_on("autoconf", type="build")
     depends_on("automake", type="build")
     depends_on("libtool", type="build")
     depends_on("m4", type="build")
+
+    def configure_args(self):
+        if self.spec.variants["shared"]:
```

Comment:
The way to check variant values for a spec is to use `self.spec.satisfies("+shared")` in this case.

See the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#testing-spec-constraints, especially note:

"The difference between the two functions is that `satisfies()` tests whether spec constraints overlap at all, while `in` tests whether a spec or any of its dependencies satisfy the provided spec."

Reply:
Favored Autotools helper.